DataScience/LogDownloader.py

```diff
@@ -144,6 +145,7 @@ def download_container(app_id, log_dir, container=None, conn_string=None, accoun
             # List all blobs and download them one by one
             Logger.info('Getting blobs list...')
             blobs = bbs.list_blobs(container)
+            blobs = sorted(list(blobs), key=lambda b: b.name, reverse=True)
```

Comment:
question + nit: This sort has the effect that we will always run evaluation against later data in the customer's time range rather than starting from the earliest point in the time range, right? Is that desired and perhaps worth a comment since that effect isn't immediately obvious?

Reply:
Discussed with Tyler. "As a customer, I would like to see how the model is performing closest to the current date. Theoretically a later model will have also learned more so will show Personalizer in a better light."

Reply:
Added comment